parse/EffectParser1.cpp

```diff
@@ -64,6 +66,8 @@ namespace {
             generate_sitrep_message
                 =    tok.GenerateSitrepMessage_
                 >    parse::label(Message_token)    >  tok.string [ _a = _1 ]
+                >  -(parse::label(Label_token)      >  tok.string [ _e = _1 ] )
+                >  -(tok.NoStringtableLookup_                     [ _f = true ] )
```

Comment:
Clearifying in the parser what is / isn't being looked up would also be better... (label or template or both?)